swagger.yaml

```
@@ -586,7 +589,7 @@ paths:
         - "application/json"
       parameters:
         - in: path
-          name: name
+          name: id
           type: string
           required: true
           description: the group's name
```

Still referencing `name`:

```suggestion
          description: the group's ID
```